rfcs/rfc0011.md

```diff
@@ -0,0 +1,76 @@
+# Command-line flag for slurping
+
+## Preamble
+
+    Author: Tomasz Konojacki <me@xenu.pl>
+    Sponsor:
+    ID: 0011
+    Status: Draft
+
+## Abstract
+
+Introduce a new command-line flag for `perl`, `-g`, which sets `$/` to `undef`,
+and thus enables slurp mode.
```

Comment:
It would be helpful to note this is to create a flag that is equivalent to `-0777` (only mented below).

Reply:
Done.